### DISCUSSION TITLE
Grafana update question.

### DISCUSSION THREAD

**Discussion by SiskoUrso** [tteck/Proxmox#1165] [Q&A] [answered]
So I didnt notice any details under the grafana install if the script should be run again in the console of the lxc or if it should be updated through grafana's documentation?

Thanks for your time.

**Comment by tteck** [comment, marked as answer]
https://github.com/tteck/Proxmox/issues/914#issuecomment-1376138884

**Reply by SiskoUrso** [reply]
thanks for the quick reply.

**Reply by malosaa** [reply]
Hi TTECK,

I have a question regarding updating Grafana,
i did run the the install 
```
bash -c "$(wget -qLO - https://github.com/tteck/Proxmox/raw/main/ct/grafana.sh)"
```

```
   ______           ____                 
  / ____/________ _/ __/___ _____  ____ _
 / / __/ ___/ __  / /_/ __  / __ \/ __  /
/ /_/ / /  / /_/ / __/ /_/ / / / / /_/ / 
\____/_/   \__,_/_/  \__,_/_/ /_/\__,_/  
 
 - Updating Grafana  -Warning: apt-key is deprecated. Manage keyring files in trusted.gpg.d instead (see apt-key(8)).
/OK
/deb [signed-by=/usr/share/keyrings/grafana.key] https://apt.grafana.com stable main
 ✓ Updated Successfully
```

It updates however it still says new version available, stays on 9.4.7.
![image](https://github.com/tteck/Proxmox/assets/13116501/50be7327-b8ac-43d0-b81e-aa20eb5c438a)

but some of the data sources not getting installed because i think the need a higher version of grafana.

|the data source i need to install is ATOM RSS.

regards
